src/Surfnet/ServiceProviderDashboard/Infrastructure/Jira/Repository/IssueRepository.php

```diff
@@ -100,7 +100,7 @@ public function findByManageIds(array $manageIds)
         foreach ($issues->issues as $issue) {
             $manageId = $issue->fields->customFields[$this->manageIdFieldName];
             if (in_array($manageId, $manageIds)) {
-                $collection[$manageId] = new Issue($issue->key, $this->issueType, $issue->fields->status);
+                $collection[$manageId] = new Issue($issue->key, $this->issueType, $issue->fields->status->id);
```

Comment:
@quartje I guessed we need the `id` from the status field. I was not able to verify this as at the moment I have no Jira test environment on my dev machine. Would you be so kind to dump the output of `var_dump($issue); die;` placed on line 102 here if the id does not turn out to be something like 'OPEN', 'RESOLVED', or 'CLOSED'.

Peace out!